projects/hslayers/src/components/add-data/url/wms/add-data-url-wms.service.ts

```diff
@@ -543,36 +554,32 @@ export class HsAddDataUrlWmsService {
    * @param group - Group layer to which add layer to
    * @param layerName - Name of layer to add. If not specified then all layers are added
    */
-  addService(
+  async addService(
```

Comment:
I did the same thing in https://github.com/hslayers/hslayers-ng/pull/2067 . I will try to rebase and resolve conflicts